sheetScript.gs

```diff
@@ -19,26 +23,23 @@ function doGet(e) {
 }
 
 function updatePlaylists(sheet) {
-  var sheetID = PropertiesService.getScriptProperties().getProperty("sheetID");
+  const sheetID = PropertiesService.getScriptProperties().getProperty("sheetID")
   if (!sheetID) onOpen()
-  if (!sheet || !sheet.toString || sheet.toString() != 'Sheet') sheet = SpreadsheetApp.openById(sheetID).getSheets()[0];
+  const spreadsheet = SpreadsheetApp.openById(sheetID)
+  if (!sheet || !sheet.toString || sheet.toString() != 'Sheet') sheet = spreadsheet.getSheets()[0];
   const MILLIS_PER_DAY = 1000 * 60 * 60 * 24;
-  var data = sheet.getDataRange().getValues();
-  var reservedTableRows = 3; // Start of the range of the PlaylistID+ChannelID data
-  var reservedTableColumns = 2; // Start of the range of the ChannelID data
-  var reservedDeleteDaysColumn = 1; // Column containing number of days before today until videos get deleted
-  var reservedTimestampCell = "F1";
+  const data = sheet.getDataRange().getValues();
+  const reservedTableRows = 3; // Start of the range of the PlaylistID+ChannelID data
+  const reservedTableColumns = 2; // Start of the range of the ChannelID data
+  const reservedDeleteDaysColumn = 1; // Column containing number of days before today until videos get deleted
+  const reservedTimestampCell = "F1";
```

Comment:
Does [this answer](https://github.com/Elijas/auto-youtube-subscription-playlist-2/pull/46/files/0e9d64782fae67b77d8e9d1d0c629f4bfd11d500#r330846411) applies here too?